build/media-suite_compile.sh

```diff
@@ -1227,6 +1227,14 @@ if [[ $mp4box = "y" ]] && do_vcs "https://github.com/gpac/gpac.git"; then
     do_checkIfExist
 fi
 
+_check=()
+if [[ $svthevc = "y" ]] && do_vcs "https://github.com/intel/SVT-HEVC.git"; then
+    do_uninstall "${_check[@]}"
+    do_basecmake -DMINGW=on -DUNIX=off -DBUILD_SHARED_LIBS=off
```

Comment:
Does `do_basecmake` already exist in the suite?

Reply:
Crap, I forgot that its only part of my cmake branch, will fix.